interface/super/edit_layout.php

```diff
@@ -1621,6 +1643,30 @@ function getNextSeq(group) {
 // jQuery stuff to make the page a little easier to use
 
 $(document).ready(function(){
+    $(function () {
+        $('.typeAddons').select2({
+            data: fldOptions,
+            theme: 'classic',
+            width:'100%',
+            placeholder: 'Select options',
```

Comment:
translate with xla()

Reply:
lol, Now I said to myself. noway will Brady catch me on escaping or translations this time........